src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/BaseQualityRankSumTest.java

```diff
@@ -24,7 +24,8 @@
  * <p>The base quality rank sum test can not be calculated for sites without a mixture of reads showing both the reference and alternate alleles.</p>
  *
  */
-public final class BaseQualityRankSumTest extends RankSumTest {
+public final class BaseQualityRankSumTest extends RankSumTest implements StandardAnnotationKind {
```

Comment:
ok, none of this this great btw because now we have a `VariantAnnotation` that does not implement `Annotation`. And if we make `VariantAnnotation` implement `Annotation` then we lose the sole purpose of the `Annotation` interface which is to have a common marker super-interface for all marker interfaces. See the problem?  Oh i hate marker interfaces.